ktxspec.adoc

```diff
@@ -1918,6 +1947,36 @@ seconds is stem:[duration / timescale].
 This metadata entry must not be used together with
 <<KTXcubemapIncomplete,`KTXcubemapIncomplete`>>.
 
+=== KTXmapRange
+When data is input from a source with 32-bit floating point data,
+such as a .exr file, and compressed to an HDR block format, such
+as ASTC HDR, BC6H or UASTC HDR, the values in the source may be
+mapped to the more limited range of the block-compressed format.
+KTX file writers may indicate the mapping by using the key
```

Comment:
This metadata is also useful for other floating-point payloads including regular `float16` types and packed formats like `rgb9e5` or `r11fg11fb10f` because they all have similarly limited ranges. The description should highlight the issue of the limited range instead of specific block-compressed formats.

FWIW, we may later add a `float64` version of this metadata since many modern GPUs support double-precision shader math.